src/config/constants.ts

```diff
@@ -71,6 +64,7 @@ export const DEFAULT_THUMBNAIL_SIZE = ThumbnailSize.Medium;
 
 export const UrlSearch = {
   KeywordSearch: 's',
+  CategorySearch: 'cat',
```

Comment:
I think leaving `category` would be fine. Should we add Google Analytics' `_gl` param here?